tutorials/plot_ecog.py

```diff
@@ -45,3 +48,15 @@
 subjects_dir = mne.datasets.sample.data_path() + '/subjects'
 fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
 mlab.view(200, 70)
+
+# If we want to make scatterplots with the figure, we can take a snapshot.
```

Comment:
I originally had it this way, but see my comment from earlier in this PR:

> It looks like strange things happen when I add a new section via sphinx gallery. If I include the header line before my comments on taking a snapshot etc (the #########) then I get an error saying that fig is of instance NoneType. If I don't include the header line, and just make them regular comments, then it runs OK.

This issue of the figure becoming `NoneType` is something that has popped up in other areas as well. Strange things seem to happen with Mayavi when I'm not running it interactively (e.g. writing tests is hard because the scene doesn't persist after the initial plot, and then becomes `None` which means we can't do things query information from it). Is this is a problem you've run into?